tools/slack/external-collaboration.md

```diff
@@ -0,0 +1,72 @@
+
+Additonal exceptions, including a small number of close TTS collaborators at GSA, may be added as full Slack members at the discretion of the Technology Director. Examples might include the Chief Information Officer or FAS Commissioner. Ask in [#admins-slack](https://gsa-tts.slack.com/messages/admins-slack) if you'd like to add someone to this category.
+
+### Dedicated channels
+
+You may want to invite partners, contractors, etc. to specific project channels to foster collaboration and asynchronous communication with the team. Projects often [create](https://get.slack.help/hc/en-us/articles/201402297-Creating-a-channel) specific channels that end with `-partners`.
+
+1. [Create a `<project>-partners` channel](https://get.slack.help/hc/en-us/articles/201402297-Creating-a-channel), if you don't want them in an existing channel.
+1. Add the partners to the channel, either:
+   - Through Slack Connect (preferred; see below)
+   - Individually, through the [user request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSfYQ-D82rIGwbCmwF3kAQERqczi5syVGq6GtmQNR6fhxRAA2Q/viewform)
+
+#### Slack Connect
+
+Slack offers [this mechanism](https://slack.com/help/articles/115004151203-Guide-to-sharing-channels-with-external-organizations) to connect channels between different workspaces. If our partner has their own Slack workspace, this is the preferred method for collaborating with them from TTS Slack, since:
+
+- It offloads user management to the partner
+- The partners can use their existing Slack accounts
+- The partner is able to retain the history
+- The history shows up in searches of their Slack workspace
+- It saves TTS money
+
+The contents of shared channels are treated the same as normal channels—see [rules](../rules/) and [records](../records/).
+
+## The public
+
+[TTS](https://www.gsa.gov/portal/category/25729) has specific channels in Slack that are open to the public to join. These channels end with `-public`. (Note: In Slack's parlance, all channels in a workspace are either `public`, allowing any full member to find/join, or `private`. However, when we say "public channels" on this page, we mean channels open to all members of the public.)
+
+Friends with `.gov`/`.mil` email addresses who aren't collaborating on a project can be invited into public channels. Fill out the [user request form](https://docs.google.com/forms/d/e/1FAIpQLSfYQ-D82rIGwbCmwF3kAQERqczi5syVGq6GtmQNR6fhxRAA2Q/viewform?usp=sf_link).
+
+Treat these public channels like you would a livestreamed event on GSA's YouTube page or other type of public meeting open to guests. Materials and documents should not be shared only in public channels; instead, they should be published on TTS websites and then linked to from the public channels so that access to Slack isn't required to see the document.
```

Comment:
>Materials and documents should not be shared 

GSA products and documents should not...

as is it could be read to not share general links/resources/etc